src/firmware/main/math/polynomial_1d.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+// Need this for size_t
+#include <stddef.h>
```

Comment:
curious why we need this?

Reply:
We use it in `polynomial_1d.c`, it's always seemed to me like the most appropriate data type for representing array size since it's >=0, and heck, "size" is in the name. Moved to the `polynomial_1d.c`.